### PR TITLE
URL click handling

### PR DESCRIPTION
**Pull request by HenriqueSPin**
Now the url passed to the notification makes the message in the panel clickable, navigating to that url.
Also added a `target` attribute that can tell the browser where to open the url: _blank, _self, ...

### CODE REVIEW

**Comment by tumainimosha** [comment]
+1

Looking forward to this feature

File: resources/js/components/NotificationMessage.vue [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <template>
   <div>
-    <div class="notification table table-fixed w-full">
+    <div class="notification table table-fixed w-full"
+         :class="{ 'cursor-pointer': this.notification.data.url }"
+         @click="handleClick">
```
Comment (4n70w4) [inline resources/js/components/NotificationMessage.vue]: May be make clickable only `notification.data.message` instead whole `notification`.

**Comment by mrvnklm** [comment]
@chrisbjr also interested, please merge 👍

**Comment by chrisbjr** [comment]
Thanks guys sorry haven't been so active lately.

This is good - I like that @HenriqueSPin has updated the roadmap as well :)

**Comment by chrisbjr** [comment]
Merged and tagged as v1.3.1

**Pull request merged**